source/fe/fe_dg_vector.cc

```diff
@@ -100,6 +101,52 @@ FE_DGBDM<dim, spacedim>::get_name() const
 }
 
 
+template <class PolynomialType, int dim, int spacedim>
+FiniteElementDomination::Domination
+FE_DGVector<PolynomialType, dim, spacedim>::compare_for_domination(
+  const FiniteElement<dim, spacedim> &fe_other,
+  const unsigned int                  codim) const
+{
+  Assert(codim <= dim, ExcImpossibleInDim(dim));
+
+  // line/face domination
+  // --------------------
+  if (codim > 0 && codim < dim)
```

Comment:
hm, isn't this also true for `codim==dim`? DG elements don't dominate anything at vertices.